pom.xml

```diff
@@ -263,8 +263,8 @@
     <project.build.resourceEncoding>UTF-8</project.build.resourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-    <javac.src.version>1.8</javac.src.version>
-    <javac.target.version>1.8</javac.target.version>
+    <javac.src.version>13</javac.src.version>
+    <javac.target.version>13</javac.target.version>
```

Comment:
Can we keep this at 1.8? I don't think we should increase the java version until we are forced to do so by one of our dependencies. There are a *lot* of applications still on java 8.

Reply:
I agree...